superset/utils/core.py

```diff
@@ -1227,11 +1227,58 @@ class TimeRangeEndpoint(str, Enum):
     UNKNOWN = "unknown"
 
 
-class ReservedUrlParameters(Enum):
+class ReservedUrlParameters(str, Enum):
     """
     Reserved URL parameters that are used internally by Superset. These will not be
     passed to chart queries, as they control the behavior of the UI.
     """
 
     STANDALONE = "standalone"
     EDIT_MODE = "edit"
+
+
+_DB_COLUMN_TYPES_NUMERIC: Tuple[str, ...] = (
```

Comment:
Why the leading underscore? Also I wanted to check are these database agnostic or should they reside in the DB engine specs?

Reply:
Now that you mention it, yes they should be moved over to db engine specs. Will do so.